src/main/kotlin/racingcar/utils/Random.kt

```diff
@@ -0,0 +1,10 @@
+package racingcar.utils
+
+import kotlin.random.Random
+
+object Random {
+
+    private val random = Random(RANDOM_SEED)
+
+    fun pickInRange(start: Int, end: Int) = random.nextInt(start, end + 1)
```

Comment:
```suggestion
    fun pickInRange(start: Int, end: Int): Int = (start..end).random()
```
으로도 간단하게 대체할수 있을거 같아요!